python/grass/jupyter/utils.py

```diff
@@ -202,6 +202,39 @@ def get_rendering_size(region, width, height, default_width=600, default_height=
     return (default_width, round(default_width * region_height / region_width))
 
 
+def dms_to_dd(dms_str):
+    dms_str = dms_str.strip()
+    direction = dms_str[-1]
+    dms_str = dms_str[:-1]
+    degrees, minutes, seconds = map(float, dms_str.split(":"))
+    dd = degrees + minutes / 60 + seconds / 3600
+    if direction in ["W", "S"]:
+        dd *= -1
+    return dd
+
+
+def get_computational_region_bb():
+    region = gs.read_command("g.region", flags="b")
```

Comment:
Use parse_command and use g.region flag g, it will parse it automatically.

Reply:
It outputs the direction and lat/lon in key, which I have to ultimately convert to key:dict pair, which would not benefit; also, with the command g, we would need to convert the directions in relation format.

Reply:
Sorry, I don't understand, isn't this what you need?
```
region = gs.parse_command("g.region", flags="bg")
region
{'ll_n': '35.80960938', 'll_s': '35.68750730', 'll_w': '-78.77462049', 'll_e': '-78.60830318', 'll_clon': '-78.69146184', 'll_clat': '35.74855834'}
```

Reply:
Sorry, I thought it was just with the "g" flag: `gs.parse_command("g.region", flags="g")` which outputs this:
`{'projection': '99',
 'zone': '0',
 'n': '228500',
 's': '215000',
 'w': '630000',
 'e': '645000',
 'nsres': '10',
 'ewres': '10',
 'rows': '1350',
 'cols': '1500',
 'cells': '2025000'}`. 
About the key:value pair thing, it was just a wrong implementation from my side.